libr/debug/debug.c

```diff
@@ -1673,3 +1673,20 @@ R_API ut64 r_debug_get_baddr(RDebug *dbg, const char *file) {
 	return 0LL;
 #endif
 }
+
+R_API void r_debug_bp_rebase(RDebug *dbg, ut64 baddr) {
+	RBreakpointItem *bp;
+	RListIter *iter;
+	r_list_foreach (dbg->bp->bps, iter, bp) {
+		bp->addr = bp->orig_addr + baddr;
+	}
+}
+
+R_API void r_debug_bp_unrebase(RDebug *dbg, ut64 baddr) {
```

Comment:
Is this function really necessary? If you "rebase" all breakpoints, to "unrebase" them just means to "rebase" with baddr=0 IMHO.

Reply:
we can do a quickfix by merging this, but this definively needs to be rethinked and done in  sync with rbin